x-pack/plugins/synthetics/public/apps/synthetics/components/monitors_page/management/monitor_list_table/columns.tsx

```diff
@@ -131,16 +128,64 @@ export function useMonitorListColumns({
       name: i18n.translate('xpack.synthetics.management.monitorList.actions', {
         defaultMessage: 'Actions',
       }),
-      render: (fields: EncryptedSyntheticsSavedMonitor) => (
-        <Actions
-          euiTheme={euiTheme}
-          configId={fields[ConfigKey.CONFIG_ID]}
-          name={fields[ConfigKey.NAME]}
-          canEditSynthetics={canEditSynthetics}
-          reloadPage={reloadPage}
-          isProjectMonitor={fields[ConfigKey.MONITOR_SOURCE_TYPE] === SourceType.PROJECT}
-        />
-      ),
+      actions: [
+        {
+          'data-test-subj': 'syntheticsMonitorEditAction',
+          isPrimary: true,
+          name: labels.EDIT_LABEL,
+          description: labels.EDIT_LABEL,
+          icon: 'pencil',
+          type: 'icon',
+          enabled: () => canEditSynthetics,
```

Comment:
Does this need to be a function that returns a boolean instead of a boolean?

Reply:
yeah it's weird eui type :/